<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1104</width>
    <height>768</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1104</width>
    <height>768</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1104</width>
    <height>768</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Order Tracker</string>
  </property>
  <property name="windowOpacity">
   <double>11.000000000000000</double>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="main_table">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>701</width>
      <height>691</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>701</width>
      <height>691</height>
     </size>
    </property>
    <property name="alternatingRowColors">
     <bool>true</bool>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <property name="gridStyle">
     <enum>Qt::SolidLine</enum>
    </property>
    <property name="sortingEnabled">
     <bool>false</bool>
    </property>
    <property name="rowCount">
     <number>0</number>
    </property>
    <column>
     <property name="text">
      <string>ID</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Date</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Owner</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Item</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Description</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Quantity</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Status</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="add_order_button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>700</y>
      <width>113</width>
      <height>37</height>
     </rect>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QPushButton" name="archive_order_button">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>700</y>
      <width>113</width>
      <height>37</height>
     </rect>
    </property>
    <property name="text">
     <string>Archive</string>
    </property>
   </widget>
   <widget class="QPushButton" name="archived_orders_button">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>700</y>
      <width>161</width>
      <height>37</height>
     </rect>
    </property>
    <property name="text">
     <string>Archived Orders</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="notes_edit">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>10</y>
      <width>401</width>
      <height>671</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="update_notes_button">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>690</y>
      <width>113</width>
      <height>37</height>
     </rect>
    </property>
    <property name="text">
     <string>Update Notes</string>
    </property>
   </widget>
   <widget class="QPushButton" name="update_order_status_button">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>700</y>
      <width>113</width>
      <height>37</height>
     </rect>
    </property>
    <property name="text">
     <string>Toggle Status</string>
    </property>
   </widget>
   <widget class="QPushButton" name="edit_order_button">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>700</y>
      <width>113</width>
      <height>37</height>
     </rect>
    </property>
    <property name="text">
     <string>Edit</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>add_order_button</tabstop>
  <tabstop>archive_order_button</tabstop>
  <tabstop>archived_orders_button</tabstop>
  <tabstop>update_notes_button</tabstop>
  <tabstop>notes_edit</tabstop>
  <tabstop>main_table</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
